### ISSUE TITLE
add 'C' to keybindings chart

### ISSUE THREAD

**Issue by gjw** [ryrobes/rvbbit#8]
C for camera?

Seems like this is probably something standard in other tools, but took me a while to figure out how to pan and zoom.

**Comment by ryrobes** [comment]
Good call. Hitting V will also toggle the zoom/pan on and off. I tend to use that, since for whatever reason holding C causes the rendering to lag out for me sometimes.
